src/test/Bau.Test.Acceptance/TaskListingFull.cs

```diff
@@ -0,0 +1,125 @@
+﻿// <copyright file="TaskListingFull.cs" company="Bau contributors">
+//  Copyright (c) Bau contributors. (baubuildch@gmail.com)
+// </copyright>
+
+namespace Bau.Test.Acceptance
+{
+    using System;
+    using System.Reflection;
+    using FluentAssertions;
+    using Support;
+    using Xbehave;
+
+    public static class TaskListingFull

```

Comment:
I think this should show the task descriptions as well, but that can come in a later PR.